profiles/resources/default.cfg

```diff
@@ -6,6 +6,7 @@ VulnSearch_____ExploitSearch (Exploits Only)_____http://www.exploitsearch.net/in
 VulnSearch_____ExploitSearch (All)_____http://www.exploitsearch.net/index.php?q=@@@PLACE_HOLDER@@@
 VulnSearch_____NVD (All)_____http://web.nvd.nist.gov/view/vuln/search-results?query=@@@PLACE_HOLDER@@@&search_type=all&cves=on
 VulnSearch_____OSVDB (All)_____http://osvdb.org/search/search?search%5Bvuln_title%5D=@@@PLACE_HOLDER@@@&search%5Btext_type%5D=alltext&search%5Bs_date%5D=&search%5Be_date%5D=&search%5Brefid%5D=&search%5Breferencetypes%5D=&search%5Bvendors%5D=&search%5Bcvss_score_from%5D=7&search%5Bcvss_score_to%5D=10&search%5Bcvss_av%5D=*&search%5Bcvss_ac%5D=*&search%5Bcvss_a%5D=*&search%5Bcvss_ci%5D=*&search%5Bcvss_ii%5D=*&search%5Bcvss_ai%5D=*&kthx=search
+VulnSearch_____ExploitSearch_____https://packetstormsecurity.com/search/?q=@@@PLACE_HOLDER@@@
```

Comment:
Name it `PacketStorm Exploit search`. See other search website names, they actually include the service name in it